src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnector.java

```diff
@@ -397,6 +447,32 @@ private Zone getZoneByTag(String zone) throws VCertException {
     return searchCertificates(Cloud.SearchRequest.findByFingerPrint(cleanFingerprint));
   }
 
+  private String[] parseZoneIdentifiers(String zone) throws VCertException {
```

Comment:
Instead this, you can create a POJO which will have ID, Project Name, and Zone Name.